#name exception
dic={"one":"1","two":"2","three":"3"}
print(dic["one"])
fp=open("satyam.txt")
